docs/source/pyfstat.rst

```diff
@@ -1,5 +1,9 @@
-pyfstat package
-===============
+.. initially generated by
+   sphinx-apidoc -f -o source ../ ../versioneer.py ../setup.py
+   (everything after the first "../" are exclude patterns)
+
+PyFstat module documentation
```

Comment:
Extra copy of the "initially generated" here is fine, but please go back to:
```suggestion
pyfstat package
```

why? compare master
![image](https://user-images.githubusercontent.com/28396587/187688669-8fcfc1f0-36a9-4215-a29f-5b8bf3db0a3e.png)

against this one
![image](https://user-images.githubusercontent.com/28396587/187688719-96690c15-ba17-4df7-a4d4-664359cae6d8.png)

Reply:
7ec24ac925a45cd5fd6983ab04675d9397de4a6c